openapi-spec/api.yaml

```
@@ -0,0 +1,811 @@
+    - Andrey Sheremeev ([sheremeev.andrey@gmail.com](mailto:sheremeev.andrey@gmail.com))
+
+
+servers:
+  - url: 'http://localhost:8081'
+    description: Local Development Server
+  - url: '/'
+    description: Current Environment
+
+
+security:
+  - bearerAuth: [ ]
```

YAMLlint (1.38.0)

[error] too many spaces inside empty brackets

(brackets)